docs/functions/data.rst

```diff
@@ -312,4 +322,3 @@ Time
    ``0``.
 
    .. versionadded:: 2.0
-
```

Comment:
It looks like your editor removed this line. Could you please add it back? The Sphinx parser gets upset if the file ends with an indented block.